Add compatibility for docker deploy

okt-limonikas opened the pull request:
## Description
This PR implements a proxy functionality for JSON logs to handle cross-origin resource sharing (CORS) issues.
This add compatibility between deploy without docker and deploy with docker, please see https://github.com/ts-factory/bublik-docker

### Key Changes
- Added a new proxy endpoint `/api/v2/logs/proxy/` that forwards requests to log files
- Added new configuration settings:
  - `SECURE_HTTP`: Flag to control HTTP/HTTPS protocol usage
  - `ENABLE_JSON_LOGS_PROXY`: Toggle for the proxy functionality

### Configuration
Two new settings have been added to `settings.py.template`:
```python
SECURE_HTTP = False
ENABLE_JSON_LOGS_PROXY = False
```

arybchik commented on .gitmodules:
```diff
@@ -0,0 +1,9 @@
+[submodule "test-environment"]
```
Sorry, but I strongly dislike it. If we want such features, we should simply add new repo for the purpose and to not pollute bublik backend with it.

okt-limonikas replied:
Seems fair. Could you create the new repo for it? I'll try to keep impact minimal then

arybchik replied:
Done, you're invited

okt-limonikas commented:
@arybchik 
I've updated description and dropped patches, moved everything to the new repo
Currently only logs not working. This PR should fix this for docker deploy
https://github.com/ts-factory/bublik-docker

arybchik commented:
> @arybchik I've updated description and dropped patches, moved everything to the new repo Currently only logs not working. This PR should fix this for docker deploy https://github.com/ts-factory/bublik-docker

Why have you decided to bypass review there?

okt-limonikas commented:
> > @arybchik I've updated description and dropped patches, moved everything to the new repo Currently only logs not working. This PR should fix this for docker deploy https://github.com/ts-factory/bublik-docker
> 
> Why have you decided to bypass review there?

My bad.
If you reset the main branch, I'll submit a proper PR for review.

ol-nata reviewed (changes requested):
1. We end commit descriptions with a period. Please add them.
2. You forgot to run the linter on *log.py*. Call `./scripts/pyformat bublik/interfaces/api_v2/log.py`.

okt-limonikas commented:
Unnecessary changes. Support for importing logs from remote storage should be handled in proxy layer
